### PR TITLE
Homework4

### PR DESCRIPTION
**Pull request by Andranik-Arakelyan**

### CODE REVIEW

File: Homework_4/commentify/src/helpers/sort.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const sort = (arr, dir = "descending") => {
```
Comment (ao-alex) [inline Homework_4/commentify/src/helpers/sort.js:1]: This part is tricky, the name suggest it should sort,  but apparently it also extends  Post object with average.
In task description I didn't notice that we can't add average rate directly to the post :), or at least we can run once on the initial array of the post, without modifying initial array, like const postsWithAverages = DUMMY_LIST.map(p=>{...p, average: "calculate it"}).

**Comment by ao-alex** [comment]
looks good, after changes you can merge it.
Next time  can you make 1 PR per homework ? here is multiple homeworks

File: Homework_4/commentify/src/helpers/sort.js (hunk 1 above)
Comment (Andranik-Arakelyan) [reply Homework_4/commentify/src/helpers/sort.js:1]: Ok, will fix. But I didn't notice where is second PR. I have 1 PR and there is just 1 homework

Comment (ao-alex) [reply Homework_4/commentify/src/helpers/sort.js:1]: there isn't 2 pr, but this PR has  Homework 2 and 3 deleted in it, you can keep them

**Pull request merged**